lib/wallaroo/core/network/connections.pony

```diff
@@ -600,12 +618,16 @@ actor Connections is Cluster
       let tcp_conn_wrapper =
         if _control_conns.contains(target_name) then
           try
+            let qqq =
```

Comment:
What is `qqq`?

Reply:
That is an artifact of the debugging spam that I'd added.  I'll remove the spam (wrt the Connections add/remove) and the need for that var.